app/pages/cif/project-revision/[projectRevision]/project-revision-create.tsx

```diff
@@ -0,0 +1,108 @@
+import { projectRevisionCreateNewQuery } from "__generated__/projectRevisionCreateNewQuery.graphql";
+import FormBase from "components/Form/FormBase";
+import {
+  projectRevisionCreateSchema,
+  projectRevisionCreateUISchema,
+} from "data/jsonSchemaForm/projectRevisionCreateSchema";
+import { JSONSchema7 } from "json-schema";
+import { useRouter } from "next/router";
+import { getProjectRevisionFormPageRoute } from "routes/pageRoutes";
+import { useCreateProjectRevisionWithRevisionType } from "mutations/ProjectRevision/createProjectRevisionWithRevisionType";
+import EmptyObjectFieldTemplate from "lib/theme/EmptyObjectFieldTemplate";
+
+const pageQuery = graphql`
+  query projectRevisionCreateNewQuery($projectRevision: ID!) {
+    session {
+      ...DefaultLayout_session
+    }
+    projectRevision(id: $projectRevision) {
+      project: projectByProjectId {
+        rowId
+        pendingProjectRevision {
+          id
+        }
+      }
+      ...TaskList_projectRevision
+    }
+  }
+`;
+
+export function ProjectRevisionCreate({
+  preloadedQuery,
+}: RelayProps<{}, projectRevisionCreateNewQuery>) {
+  const { session, projectRevision } = usePreloadedQuery(
+    pageQuery,
+    preloadedQuery
+  );
+  const taskList = <TaskList projectRevision={projectRevision} mode={"view"} />;
+
+  const router = useRouter();
+  const [createProjectRevision, isCreatingProjectRevision] =
+    useCreateProjectRevisionWithRevisionType();
+
+  const handleCreateRevision = ({ formData }) => {
+    console.log(formData.revisionType);
+    createProjectRevision({
+      variables: {
+        projectId: projectRevision.project.rowId,
+        revisionType: formData.revisionType,
+      },
+      onCompleted: (response) => {
+        router.push(
+          getProjectRevisionFormPageRoute(
+            response.createProjectRevisionWithRevisionType.projectRevision.id,
+            0
+          )
+        );
+      },
+    });
+  };
+  return (
+    <>
+      <DefaultLayout session={session} leftSideNav={taskList}>
+        <div>
+          <FormBase
+            id="ProjectRevisionCreateForm"
+            schema={projectRevisionCreateSchema as JSONSchema7}
+            uiSchema={projectRevisionCreateUISchema}
+            onSubmit={handleCreateRevision}
+            ObjectFieldTemplate={EmptyObjectFieldTemplate}
+          >
+            <Button
+              type="submit"
+              size="small"
+              disabled={
+                isCreatingProjectRevision ||
+                projectRevision?.project.pendingProjectRevision !== null
+              }
+            >
+              New Revision
+            </Button>
+          </FormBase>
+        </div>
+      </DefaultLayout>
+      <style jsx>{`
+        div :global(.radio) {
+          padding: 5px;
+        }
+        div :global(.radio):first-child {
+          margin-top: 25px;
+        }
+        div :global(input[type="radio"]) {
+          margin-right: 10px;
+        }
```

Comment:
This should go in the rjsf field template: `FieldTemplate.tsx` instead, so all forms get the same styling